src/test/java/org/prebid/server/bidder/gothamads/GothamAdsBidderTest.java

```diff
@@ -0,0 +1,235 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.gotthamads.GothamAdsBidder;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.gothamads.GothamAdsImpExt;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import static java.util.function.Function.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.prebid.server.bidder.model.BidderError.Type;
+import static org.prebid.server.bidder.model.BidderError.badServerResponse;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.banner;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.video;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.xNative;
+import static org.prebid.server.util.HttpUtil.ACCEPT_HEADER;
+import static org.prebid.server.util.HttpUtil.APPLICATION_JSON_CONTENT_TYPE;
+import static org.prebid.server.util.HttpUtil.CONTENT_TYPE_HEADER;
+import static org.prebid.server.util.HttpUtil.USER_AGENT_HEADER;
+import static org.prebid.server.util.HttpUtil.X_FORWARDED_FOR_HEADER;
+import static org.prebid.server.util.HttpUtil.X_OPENRTB_VERSION_HEADER;
+import static org.prebid.server.util.HttpUtil.headers;
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+
+public class GothamAdsBidderTest extends VertxTest {
+
+    private static final String ENDPOINT_URL = "https://test-url.com/?pass={{AccountId}}";
+    private static final String OPEN_RTB_VERSION = "2.5";
+    private static final String DEVICE_UA = "ua";
+    private static final String IPV_6_HEADER_VALUE = "ipv6";
+    private static final String IP_HEADER_VALUE = "ip";
+
+    private GothamAdsBidder sut;
+
+    @BeforeEach
+    public void setUp() {
+        sut = new GothamAdsBidder(ENDPOINT_URL, jacksonMapper);
+    }
+
+    @Test
+    public void shouldFailOnBidderCreation() {
+        assertThatIllegalArgumentException().isThrownBy(() -> new GothamAdsBidder("invalid_url", jacksonMapper));
+    }
+
+    @Test
+    public void shouldReturnErrorWhileMakingHttpRequestsWhenRequestHasInvalidImpression() {
+        // given
+        final var invalidExt = (ObjectNode) mapper.valueToTree(ExtPrebid.of(null, mapper.createArrayNode()));
+        final var bidRequest = givenBidRequest(impBuilder -> impBuilder.ext(invalidExt));
```

Comment:
Please, add empty lines between given - when - then blocks. Also, please, check for similar occurences.